virtualizarr/tests/test_readers/test_dmrpp.py

```diff
@@ -1,22 +1,353 @@
+from pathlib import Path
+from xml.etree import ElementTree as ET
+
+import numpy as np
 import pytest
 import xarray as xr
+import xarray.testing as xrt
 
 from virtualizarr import open_virtual_dataset
+from virtualizarr.manifests.manifest import ChunkManifest
+from virtualizarr.readers.dmrpp import DMRParser
 from virtualizarr.tests import network
 
 urls = [
     (
-        "netcdf4",
-        "https://github.com/OPENDAP/bes/raw/3e518f6dc2f625b0b83cfb6e6fd5275e4d6dcef1/modules/dmrpp_module/data/dmrpp/chunked_threeD.h5",
-        "dmrpp",
-        "https://github.com/OPENDAP/bes/raw/3e518f6dc2f625b0b83cfb6e6fd5275e4d6dcef1/modules/dmrpp_module/data/dmrpp/chunked_threeD.h5.dmrpp",
+        "https://its-live-data.s3-us-west-2.amazonaws.com/test-space/cloud-experiments/dmrpp/20240826090000-JPL-L4_GHRSST-SSTfnd-MUR25-GLOB-v02.0-fv04.2.nc",
+        "https://its-live-data.s3-us-west-2.amazonaws.com/test-space/cloud-experiments/dmrpp/20240826090000-JPL-L4_GHRSST-SSTfnd-MUR25-GLOB-v02.0-fv04.2.nc.dmrpp",
     )
+    # TODO: later add MUR, SWOT, TEMPO and others by using kerchunk JSON to read refs (rather than reading the whole netcdf file)
 ]
 
 
+@pytest.fixture
+def basic_dmrpp() -> DMRParser:
+    xml_str = """<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
I assume you mean so that the `<?xml?` tag can be moved to the next line for readbility? The problem is that this now adds a newline and `textwrap.dedent` keeps that newline

Reply:
You could potentially use a backslash like the example here? https://docs.python.org/3/library/textwrap.html#textwrap.dedent

This isn't very important though, feel free to ignore.

Reply:
Oh nice, that works then. Added